kerrokantasi/settings/base.py

```diff
@@ -1,12 +1,30 @@
 import os
+
+import environ
+import raven
+
 gettext = lambda s: s
 
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
 
-SECRET_KEY = '52k^*)c*bz9t0lzsf_$a+jl3zcy6re!gnw77__)y(#v91-p%tp'
-DEBUG = True
+env = environ.Env()
 
-ALLOWED_HOSTS = []
+DEBUG = env.bool('DEBUG', default=True)
+SECRET_KEY = env.str('SECRET_KEY', default=('x' if DEBUG else ''))
+ALLOWED_HOSTS = env.list('ALLOWED_HOSTS', default=[])
+ADMINS = env.list('ADMINS', default=[])
+DATABASES = {
+    'default': env.db('DATABASE_URL', default='postgis:///kerrokantasi')
+}
+SENTRY_DSN = env.str('SENTRY_DSN', default='')
+DEMOCRACY_UI_BASE_URL = env.str('DEMOCRACY_UI_BASE_URL', default='http://localhost:8086')
+
+JWT_AUTH = {
+    'JWT_SECRET_KEY': env.str('JWT_SECRET_KEY', default='kerrokantasi'),
+    'JWT_AUDIENCE': env.str('JWT_AUDIENCE', default='kerrokantasi')
+}
+
+### Settings below do not usually need changing
```

Comment:
This is something of a moot comment since you're probably not supposed to modify this file locally anyway, just use envvars.

Reply:
Still this file serves as a template, when using local_settings (which is still wanted)

Reply:
The developer might override these in `local_settings.py`, though.